src/config/images.js

```diff
@@ -14,6 +14,14 @@ export const MENTORSHIP_IMG =
   'https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/community%2Fmentorship.jpg?alt=media'
 export const LONDON =
   'https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/london.jpg?alt=media'
+
+// export const LONDON  = () => (
```

Comment:
@shakersinthedark we try not to keep commented code in the projects. Please remove it or leave a comment with the reason why that comment should be there

Reply:
Oh shizzle! Totally left that there by accident. i'll remove that ;)